src/whatsnew/3.0.rst

```diff
@@ -0,0 +1,398 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _release/3.0.x:
+
+============
+3.0.x Branch
+============
+
+.. contents::
+    :depth: 1
+    :local:
+
+.. _release/3.0.x/upgrade:
+
+Upgrade Notes
+=============
+
+.. rst-class:: open
+
+* :ghissue:`2228`: The default maximum document size has been reduced to 8MB. This means
+  that databases with larger documents will not be able to replicate into CouchDB 3.0
+  correctly without modification. This change has been made in preparation for
+  anticipated hard upper limits on document size imposed by CouchDB 4.0. For 3.x,
+  the max document size setting can be relaxed via the ``[couchdb] max_document_size``
+  config setting.
+* :ghissue:`2228`: The default database sharding factor ``q`` has been reduced to 2 by
+  default. This, combined with automated database resharding (see below), is a better
+  starting place for new CouchDB databases. As in CouchDB 2.x, specify ``?q=##`` to
+  change the value upon database creation if desired. The default can be changed
+  via the config ``[cluster] q`` setting.
+* :ghissue:`1523`, :ghissue:`2092`, :ghissue:`2336`: The "node-local" HTTP interface,
+  by default exposed on port 5986, has been removed. All functionality previously
+  available at that port is now available on the main, clustered interface (by default,
+  port 5984). Examples:
+
+  * ``GET /_node/{nodename}/_stats``
+  * ``GET /_node/{nodename}/_system``
+  * ``GET /_node/{nodename}/_restart``
+  * ``GET /_node/{nodename}/_all_dbs``
+  * ``GET /_node/{nodename}/_uuids``
+  * ``GET /_node/{nodename}/_config``
+  * ``GET /_node/{nodename}/_config/couchdb/uuid``
+  * ``GET /_node/{nodename}/_nodes/_changes?include_docs=true``
+  * ``GET /_node/{nodename}/{db}/{ddoc}/_info``
+  * ``PUT /_node/{nodename}/_dbs/{dbname}``
+
+  ...and so on. Documentation has been updated to reflect this change. [TODO]
+
+* :ghissue:`2389`: CouchDB 3.0 now requires a server admin user to be defined at
+  startup, or will print an error message and exit. (The Admin Party is now over.)
```

Comment:
needs link to a sad trombone gif :D https://media.giphy.com/media/4GXOphKW64BZZdh0Au/giphy.gif

Reply:
added cute cat picture with CC-BY-NC license, feel free to bikeshed